docs/CONTRIBUTING.md

```diff
@@ -285,7 +285,7 @@ by typing `\q`) without any errors to connect to your running Postgres server.
 On all platforms, install through `cargo` by running:
 
 ```
-cargo install diesel_cli --no-default-features --features postgres
+cargo install diesel_cli --no-default-features --features postgres --version 1.2.0
```

Comment:
```suggestion
cargo install diesel_cli --no-default-features --features postgres --version 1
```

I would assume this might also work and install the latest compatible 1.x release?

Reply:
I corrected it.